include/amici/defines.h

```diff
@@ -180,7 +180,8 @@ enum class NonlinearSolverIteration {
 /** Sensitivity computation mode in steadyStateProblem */
 enum class SteadyStateSensitivityMode {
```

Comment:
I think we need some documentation of the behavior for the different values, for example in the function `Model::setSteadyStateSensitivityMode()`. https://github.com/AMICI-dev/AMICI/blob/master/python/examples/example_constant_species/ExampleEquilibrationLogic.ipynb also needs to be updated.

Reply:
Yeah, this notebook definitely  needs to be updated.